barpyrus/colors.py

```diff
@@ -0,0 +1,34 @@
+# gruvbox pallete
+BG0_H = '#1d2021'
+BG0_S = '#32302f'
+BG0 = '#282828'
+BG1 = '#3c3836'
+BG2 = '#504945'
+BG3 = '#665c54'
+BG4 = '#7c6f64'
+FG0 = '#fbf1c7'
+FG1 = '#ebdbb2'
+FG2 = '#d5c4a1'
+FG3 = '#bdae93'
+FG4 = '#a89984'
+FG = FG1
+BG = BG0
+RED_DARK = '#cc241d'
+GREEN_DARK = '#98971a'
+YELLOW_DARK = '#d79921'
+BLUE_DARK = '#458588'
+PURPLE_DARK = '#b16286'
+AQUA_DARK = '#689d6a'
+GRAY_DARK = '#a89984'
+ORANGE_DARK = '#d65d0e'
+RED_LIGHT = '#fb4934'
+GREEN_LIGHT = '#b8bb26'
+YELLOW_LIGHT = '#fabd2f'
+BLUE_LIGHT = '#83a598'
+PURPLE_LIGHT = '#d3869b'
+AQUA_LIGHT = '#8ec07c'
+GRAY_LIGHT = '#928374'
+ORANGE_LIGHT = '#fe8019'
+
+def col_fmt(color):
+    return '%{F\\' + color + '}'
```

Comment:
This is not a general barpyrus color formatting but it only works in conky, so `col_fmt` needs to go to `conky.py`.

Reply:
Ah okay!

Reply:
Done